packages/commonwealth/client/scripts/views/pages/AdminPanel/RPCEndpointTask.tsx

```diff
@@ -75,23 +73,14 @@ const RPCEndpointTask = () => {
     500,
   );
 
-  const { data: communityLookupData, isLoading: isLoadingCommunityLookupData } =
-    useGetCommunityByIdQuery({
-      id: debouncedCommunityLookupId || '',
-      enabled: !!debouncedCommunityLookupId,
-    });
-
-  const communityNotFound =
-    !isLoadingCommunityLookupData &&
-    (!communityLookupData ||
-      Object.keys(communityLookupData || {})?.length === 0);
-
-  const communityNotChain = communityLookupData?.type !== ChainType.Chain;
+  const { data: communityLookupData } = useGetCommunityByIdQuery({
+    id: debouncedCommunityLookupId || '',
+    enabled: !!debouncedCommunityLookupId,
```

Comment:
this query won't run since we never `setRpcEndpointCommunityId` with a valid id. Also it will lead to no update calls as that logic depends on this data fetching.